compiler/rustc_codegen_cranelift/src/base.rs

```diff
@@ -858,6 +858,15 @@ fn codegen_stmt<'tcx>(
                             lval.write_cvalue(fx, val);
                             return;
                         }
+                        NullOp::ContractChecks => {
+                            let val = fx.tcx.sess.contract_checks();
+                            let val = CValue::by_val(
+                                fx.bcx.ins().iconst(types::I8, i64::try_from(val).unwrap()),
```

Comment:
```suggestion
                                fx.bcx.ins().iconst(types::I8, i64::from(val)),
```

There is an `From<bool> for i64` impl.